pkg/metrics/prometheus.go

```diff
@@ -98,8 +108,23 @@ func NewPrometheus(name string) MetricCollector {
 		gauges:     make(map[string]*prometheus.GaugeVec),
 		counters:   make(map[string]*prometheus.CounterVec),
 		name:       name,
+		registry:   &prometheus.DefaultRegisterer,
 	}
 	p.histograms[prefix] = functionDuration
+	// Register the http_server_request_duration_seconds histogram
+	responseTimeHistogram := prometheus.NewHistogramVec(prometheus.HistogramOpts{
+		Namespace: p.name,
+		Name:      "http_server_request_duration_seconds",
+		Help:      "Histogram of response time for handler in seconds",
+		Buckets:   []float64{.005, .01, .025, .05, .1, .25, .5, 1, 2.5, 5, 10},
+	}, []string{"route", "method", "status_code"})
+	p.histograms["http_server_request_duration_seconds"] = responseTimeHistogram
+
+	p.histograms["http_server_request_duration_seconds"] = responseTimeHistogram
```

Comment:
Is it intended for these lines to be duplicated?

Reply:
Removed the duplicate.